docs/blog/introducing-weps.rst

```diff
@@ -0,0 +1,81 @@
+Introducing Write the Docs Enhancement Proposals
+================================================
+
+Write the Docs has been growing each year for the past 7,
+and the scope of the community has expanded dramatically.
+We started with a single conference,
+and now we have multiple conferences,
+over 50 meetups around the world,
+multiple online spaces,
+and many other ongoing projects.
+
+We have often followed open source in our approach to things,
+keeping a loose governance model and mostly letting the work speak for itself.
+However,
+we've been noticing a number of growing pains with this approach.
+It makes it much harder for new people to understand and join the community in leadership positions,
+since they don't understand how the community actually works.
+Even folks doing a lot of work in the community are often lost or confused when it comes to other ongoing projects.
+
+All organizations have power structures,
+some are just not documented. [#]_
+Instead of trying to reinvent the wheel,
+we've once again taken an idea from the open source ecosystem:
+Write the Docs Enhancement Proposals,
+or WEP's.
+
+What is a Write the Docs Enhancement Proposal (WEP)
+---------------------------------------------------
+
+WEP's are a formalized process for making major policy changes in the community.
+Previously there was fuzzy consensus on a Slack channel or at a conference that defined our governace.
+Going forward, there will be a written record that people have had time to review and comment on.
+Importantly, 
+there will also be records of the conversation that lead to a change.
+
+We hope that WEP's will make it explicit how major changes are made inside the community.
+New contributors can propose them with a well-defined process,
+but can also see WEP's proposed by existing contributors.
+WEP's are the first step towards having a more open and defined model for governance,
+but certainly not the last.
+All further governance changes are planned to go through the WEP process,
+so it is truly foundational for ongoing work on how the community is managed.
+
+What is the WEP process?
+------------------------
+
+The WEP process is defined in `WEP0`_,
+the first ever WEP.
+The process is currently undergoing review,
+following the draft WEP process.
+This is a bit circular,
+but we hope that it will showcase how the process works as we adopt it.
+
+The high-level overview of the WEP process is as follows,
+from the perspective of someone proposing a WEP:
+
+* Talk to people about your idea, and get feedback from other community members
+* Draft and submit a Pull Request on the WEP GitHub Repository, it is now a **draft**
+* The **WEP team** then reviews the Draft WEP to make sure it passes a few tests:
```

Comment:
Maybe something more like 

... to make sure it has the following characteristics